src/godot-api/builtins.md

```diff
@@ -84,8 +84,18 @@ Godot provides three string types: `String` ([`GString`][api-gstring] in Rust),
 `GString` is used as a general-purpose string, while `StringName` is often used for identifiers like class or action names.
 The idea is that `StringName` is cheap to construct and compare.[^string-name-Rust]
 
-These types all support `From` traits to convert to/from Rust `String`, and from `&str`. You can thus use `"some_string".into()` expressions.
-If you need more explicit typing, use `StringName::from("some_string")`.
+When working with Godot APIs, you can pass references to the parameter type (e.g. &GString), as well as Rust strings `&str`, and `&String`.
```

Comment:
```suggestion
When working with Godot APIs, you can pass references to the parameter type (e.g. `&GString`), as well as Rust strings `&str`, and `&String`.
```
I missed the backticks there, you should be able to commit directly via web button.